x-pack/plugins/cases/server/routes/api/user_actions/find_user_actions.ts

```diff
@@ -6,11 +6,11 @@
  */
 
 import { schema } from '@kbn/config-schema';
+import type { userActionApiV1 } from '../../../../common/types/api';
```

Comment:
Ah, this is what you mean right? (+ `get_case_user_action_stats` and `get_all_user_actions`).

Reply:
Alright, so if these are already tied to a version should for example the `find_user_actions` integration tests use v1 everything?